tests/server/commons/test_records_dao.py

```diff
@@ -26,6 +26,7 @@ def test_raise_proper_error():
         dao.search_records(
             dataset=BaseDatasetDB(
                 name="mock-notfound",
+                workspace="ee",
```

Comment:
If `ee` is a random string maybe we should use more contextual strings like:

```suggestion
                workspace="workspace-name",
```

Correct me if I'm wrong.

Reply:
+1